nuancier/static/upload.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * HTML5 Drag and Drop uploader
+ *
+ * This file has been adapted for the needs of pagures from the work done by
+ * Noah Petherbridge in https://github.com/kirsle/flask-multi-upload/
```

Comment:
Maybe add a note in this comment that says the file was originally from ``flask-multi-upload``, but was then copied and modified for ``pagure``, but was then again copied and modified for ``nuancier``.